### PR TITLE
documentation for ChannelPO, HarmoniaApplication, HarmoniaConstants, UserClient

### PR DESCRIPTION
**Pull request by Jihau**

### CODE REVIEW

File: Frontend/src/main/java/com/harmonia/HarmoniaApplication.java [hunk 1]
```diff
@@ -13,7 +13,7 @@
 /**
  * Class representing the running class of the Harmonia application.
  *
- * @author OTP1 Group 3: Johanna Toivanen, Mohammed Al-Jewari, Sampo Savolainen, Jesper Ojala
```
Comment (psolizrodriguez) [inline Frontend/src/main/java/com/harmonia/HarmoniaApplication.java:16]: Harmonia Team :frog:

**Review by psolizrodriguez — COMMENTED** [review]
Hmm

File: Frontend/src/main/java/com/harmonia/client/UserClient.java [hunk 2]
```diff
@@ -44,12 +68,11 @@ public ResponseEntity<Void> removeUser(int userId) {
         return restTemplate.exchange(USERS_DELETE_URL, HttpMethod.DELETE, request, Void.class, urlParameters);
     }
 
-    public UserPO editPassword(int target, String newpassword) {
```
Comment (mohammja) [inline Frontend/src/main/java/com/harmonia/client/UserClient.java:47]: Is this one old method that we are not using anymore?

Comment (Jihau) [reply Frontend/src/main/java/com/harmonia/client/UserClient.java:47]: there was two methods for editPassword. This one was missing the urlParameters so it was doing nothing. That is why it is now deleted.

**Review by mohammja — APPROVED** [review]

File: Frontend/src/main/java/com/harmonia/HarmoniaApplication.java (hunk 1 above)
Comment (Jihau) [reply Frontend/src/main/java/com/harmonia/HarmoniaApplication.java:16]: magic, I changed it before you left your comment.

**Pull request merged**